NYTimes.com.js

```diff
@@ -51,13 +61,14 @@ function detectWeb(doc, url) {
 
 
 function scrape(doc, url) {
-	var type = detectWeb(doc, url);
+	var type = detectWebHere(doc, url);
 	var translator = Zotero.loadTranslator('web');
 	// Embedded Metadata
 	translator.setTranslator('951c027d-74ac-47d4-a107-9c3069ab7b48');
 	//translator.setDocument(doc);
 	
 	translator.setHandler('itemDone', function (obj, item) {
+		item.itemType = type;
```

Comment:
This should AFAIK not make any difference. However, I tried it out during testing different attempts (for the problems with detectWeb) and it seems I didn't change it back.